src/app/result/result.component.ts

```diff
@@ -63,6 +65,8 @@ export class ResultComponent {
     this.router.navigate(['/front']);
   }
 
-
+  imageUrlNotCached(url) {
+    return url + '?' + this.uuid;
+  }
```

Comment:
Have you tested that this actually produces a different URL within one session?
Try it with a pure function ✨  (and a test)